tests/queries/0_stateless/01889_sqlite_read_write.sh

```diff
@@ -73,6 +73,8 @@ ${CLICKHOUSE_CLIENT} --query="CREATE TABLE sqlite_table3 (col1 String, col2 Int3
 ${CLICKHOUSE_CLIENT} --query='SHOW CREATE TABLE sqlite_table3;' | sed -r 's/(.*SQLite)(.*)/\1/'
 ${CLICKHOUSE_CLIENT} --query="INSERT INTO sqlite_table3 VALUES ('line\'6', 6);"
 ${CLICKHOUSE_CLIENT} --query="INSERT INTO sqlite_table3 VALUES (NULL, 7);"
+${CLICKHOUSE_CLIENT} --query="INSERT INTO sqlite_table3 VALUES ('\'', 8);"
+${CLICKHOUSE_CLIENT} --query="INSERT INTO sqlite_table3 VALUES ('\\\\', 9);" # Escaped backslash = \\, but we're in bash - so \\\\
```

Comment:
You changed it for FixedString, but the table uses String here.

Reply:
Thanks, I'll add an appropriate test.